eng/AfterSigning.targets

```diff
@@ -8,4 +8,24 @@
       Text="$(RazorExtensionVSIXName) was not generated."
       Condition="!Exists('$(VSSetupDir)$(Configuration)\$(RazorExtensionVSIXName)')" />
   </Target>
+
+  <Target Name="_ZipLanguageServerBinaries" AfterTargets="Pack"  Condition=" '$(OS)' == 'Windows_NT' ">
+    <!-- Build a .zip file from each platform's directory and write it to 'artifacts' -->
+    <PropertyGroup>
+      <RidsPublishDir>$(ArtifactsDir)LanguageServer\$(Configuration)\</RidsPublishDir>
```

Comment:
Is this directory intended to be published? Will likely require some changes to eng/Publishing.props before that will work.  Not sure if that file has merged into 'release/3.1' yet.  I'm not positive about the magic incantation to publish daily builds into a VS input feed but this should be included in BlobArtifacts, like our installers.

Reply:
Yup it is. Thanks for the insight Doug!

Reply:
To publish automatically i.e. without the release pipeline (which is required for validation of private changes), put the blobs in `$(ArtifactsDir)Installers` and mimic AspNetCore's
- [Publishing.props](https://github.com/aspnet/AspNetCore/blob/d19093ecbb91a0dd1d7b2cb50234ec590baa730c/eng/Publishing.props#L11-L55)
- [ci.yml](https://github.com/aspnet/AspNetCore/blob/d19093ecbb91a0dd1d7b2cb50234ec590baa730c/.azure/pipelines/ci.yml#L628)
  - azure-pipelines.yml in this repo

If you also need to push packages to the VS package feed, will need another [ci.yml](https://github.com/aspnet/AspNetCore/blob/d19093ecbb91a0dd1d7b2cb50234ec590baa730c/.azure/pipelines/ci.yml#L167-L175) / azure-pipelines.yml change.

Reply:
In any case, rename this directory to `Installers` to match Arcade conventions.